pkg/volume/plugins.go

```diff
@@ -121,6 +122,9 @@ type VolumeHost interface {
 	// the provided spec.  See comments on NewWrapperBuilder for more
 	// context.
 	NewWrapperCleaner(spec *Spec, podUID types.UID, mounter mount.Interface) (Cleaner, error)
+
+	//Get cloud provider from kubelet
+	GetCloudProvider() cloudprovider.Interface
```

Comment:
@saad-ali This is an extension to the VolumeHost interface, but I think I am OK with it.  Your thoughts?